### DISCUSSION TITLE
科掌门卷帘门控制器没实体

### DISCUSSION THREAD

**Discussion by csky888** [XiaoMi/ha_xiaomi_home#323] [Ideas]
![image](https://github.com/user-attachments/assets/b0190fb7-7e59-4c65-b122-9a88a834e615)
希望能添加实体，非常感谢

**Comment by JimmyKmi** [comment]
图片没有上传成功，请提供设备model

**Reply by csky888** [reply]
Aice.motor.kzmu3
希望能加入实体

**Reply by JimmyKmi** [reply]
可以自行到 https://mijia.wiki/products/aice.motor.kzmu3 检索，参考“完整自动化”，若有完整自动化但集成没有可以提交，若“完整自动化”中缺少功能则需要联系场景新增功能。

**Comment by csky888** [comment]
![IMG_4523](https://github.com/user-attachments/assets/6583e74e-36bf-4d98-8f76-c90ff59caa67)
![426dfb8c93daa508ca366b3ad41a1703](https://github.com/user-attachments/assets/835e685e-ee10-4aab-8a0e-eb393aca450b)
auto有实体用不了

**Reply by csky888** [reply]
希望在xiaomi home加入实体操作，比较方便使用，谢谢
